fn main() -> Result<(), Box<dyn std::error::Error>> {
    vipe_rs::run()
}
